app/reducers/channels.js

```diff
@@ -386,6 +387,15 @@ const filtersSelector = state => state.channels.filters
 const filterSelector = state => state.channels.filter
 const nodesSelector = state => state.network.nodes
 
+const displayNodeName = (channel, nodes) => {
+  const node = find(nodes, n => channel.remote_pubkey === n.pub_key)
```

Comment:
I think you can use regular `find` here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find